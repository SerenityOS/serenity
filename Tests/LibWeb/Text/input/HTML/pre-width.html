<script src="../include.js"></script>
<script>
    test(() => {
        let testCounter = 1;
        function testPart(part) {
            println(`${testCounter++}. ${JSON.stringify(part())}`);
        }

        // 1. Set the pre element width attributes
        testPart(() => {
            const pre = document.createElement('pre');
            pre.width = 20;
            return pre.width;
        });

        // 2. Set the pre element width attributes with invalid data
        testPart(() => {
            const pre = document.createElement('pre');
            pre.width = 'dsafds';
            return pre.width;
        });

        // 3. Get the pre element width attribute default value
        testPart(() => {
            const pre = document.createElement('pre');
            return pre.width;
        });
    });
</script>
