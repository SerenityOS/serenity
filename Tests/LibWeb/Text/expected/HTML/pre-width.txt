1. 20
2. 0
3. 0
